app/src/main/java/nerd/tuxmobil/fahrplan/congress/alarms/AlarmReceiver.java

```diff
@@ -29,6 +31,7 @@
 
     private static final String LOG_TAG = "AlarmReceiver";
 
+    @SuppressWarnings("java:S2245")
```

Comment:
What's this doing? A Google search turned up nothing useful.

Reply:
I was testing this branch against SonarQube. Here is the rule specification: https://jira.sonarsource.com/browse/RSPEC-2245.

Reply:
Obsolete in b0d4b62a01a9cced5e158e52029f538e469fc89c.